src/pages/Autolending/CheckList.vue

```diff
@@ -1,18 +1,32 @@
 <template>
-	<ul class="check-list" :class="{ 'use-columns': useColumns }">
-		<li
-			:key="id"
-			v-for="{ id, name, selected } in items"
+	<div>
+		<button
+			v-if="items.some(x => !x.selected)"
+			@click="emitAll(true)"
 		>
-			<kv-checkbox
-				:id="`${name}-${id}` | changeCase('paramCase')"
-				:checked="selected"
-				@change="$emit('change', $event, id);"
+			Select All
+		</button>
+		<button
+			v-if="items.some(x => x.selected)"
```

Comment:
This works, but it would be better style to put the array method in a computed property and check for that value in the v-if.